### PR TITLE
Feature/tsc

### PR DESCRIPTION
**Pull request by mauceballos**
add test case for TSC module and Tabs.

### CODE REVIEW

File: test/performance/modules/PCI-DSS-module-dashboard.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    await commands.wait.bySelector('a[href$="/app/wazuh"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('a[href$="/app/wazuh"]')
+    
+    //Wait for an Wazuh home page component to be loaded
+    await commands.wait.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]', WAIT_TIMEOUT)
+    await commands.wait.byCondition("!isNaN(parseInt(document.querySelector('.statWithLink').innerHTML))", WAIT_TIMEOUT)
+    await commands.click.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]')
+
+    // Start collecting metrics
+    await commands.measure.start('PCI DSS module - Dashboard')
```
Comment (Mayons95) [inline test/performance/modules/PCI-DSS-module-dashboard.js:21]: The metrics should start between line 17 & 18.
The test ignores some request if the metrics starts pass that point
Screenshot from 2022-08-04 07-53-54
png)

File: test/performance/modules/PCI-DSS-module-dashboard.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    await commands.click.bySelector('button[data-test-subj="toggleNavButton"]')
+    await commands.wait.bySelector('a[href$="/app/wazuh"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('a[href$="/app/wazuh"]')
+    
+    //Wait for an Wazuh home page component to be loaded
+    await commands.wait.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]', WAIT_TIMEOUT)
+    await commands.wait.byCondition("!isNaN(parseInt(document.querySelector('.statWithLink').innerHTML))", WAIT_TIMEOUT)
+    await commands.click.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"PCI DSS")]')
+
+    // Start collecting metrics
```
Comment (Mayons95) [inline test/performance/modules/PCI-DSS-module-dashboard.js:20]: Metrics must start between lines 17 and 18.
The test ignores some requests if the metrics start past that point
Screenshot from 2022-08-04 07-53-54
Screenshot from 2022-08-04 07-54-32

File: test/performance/modules/TSC-module-dashboard.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    await commands.wait.bySelector('a[href$="/app/wazuh"]', WAIT_TIMEOUT)
+    await commands.click.bySelector('a[href$="/app/wazuh"]')
+    
+    //Wait for an Wazuh home page component to be loaded
+    await commands.wait.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"TSC")]', WAIT_TIMEOUT)
+    await commands.wait.byCondition("!isNaN(parseInt(document.querySelector('.statWithLink').innerHTML))", WAIT_TIMEOUT)
+    await commands.click.byXpath('//*[contains(@class,"euiTitle euiTitle--small euiCard__title")]//*[contains(text(),"TSC")]')
+
+    // Start collecting metrics
+    await commands.measure.start('TSC module - Dashboard')
```
Comment (Mayons95) [inline test/performance/modules/TSC-module-dashboard.js:21]: Same comment as before

**Review by Mayons95 — CHANGES_REQUESTED** [review]
Please see the comments

File: test/performance/modules/PCI-DSS-module-dashboard.js (hunk 2 above)
Comment (mauceballos) [reply test/performance/modules/PCI-DSS-module-dashboard.js:19]: Done

File: test/performance/modules/TSC-module-dashboard.js (hunk 3 above)
Comment (mauceballos) [reply test/performance/modules/TSC-module-dashboard.js:20]: Done

**Review by Mayons95 — APPROVED** [review]

**Comment by Mayons95** [comment]
CR :heavy_check_mark: 
TR :heavy_check_mark:

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark: 
Test: :heavy_check_mark:

**Comment by mauceballos** [comment]
**GDPR**  Module

>       - Control
>       - Dashboard
>       - Events
>       
> <details>
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-GDPR-module-controls-js-2022-08-08-10-25-49-pages-html-2022-08-08-10_29_07
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-GDPR-module-controls-js-2022-08-08-10-25-49-pages-localhost-GDPR-module-Controls-index-html-2022-08-08-10_29_22
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-GDPR-module-controls-js-2022-08-08-10-25-49-pages-localhost-GDPR-module-Dashboard-index-html-2022-08-08-10_29_38
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-GDPR-module-controls-js-2022-08-08-10-25-49-pages-localhost-GDPR-module-Events-index-html-2022-08-08-10_29_52
> 
>  </details>



**PCI-DSS**  Module

>       - Control
>       - Dashboard
>       - Events
> 
> <details>
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-PCI-DSS-module-controls-js-2022-08-08-10-47-01-pages-html-2022-08-08-10_50_47
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-PCI-DSS-module-controls-js-2022-08-08-10-47-01-pages-localhost-PCI-DSS-module-Controls-index-html-2022-08-08-10_50_38
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-PCI-DSS-module-controls-js-2022-08-08-10-47-01-pages-localhost-PCI-DSS-module-Dashboard-index-html-2022-08-08-10_50_28
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-PCI-DSS-module-controls-js-2022-08-08-10-47-01-pages-localhost-PCI-DSS-module-Events-index-html-2022-08-08-10_50_15
> 
>  </details>

 

**NIST** Module

>       - Control
>       - Dashboard
>       - Events
> 
> <details>
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-NIST-module-controls-js-2022-08-08-10-36-39-pages-html-2022-08-08-10_40_14
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-NIST-module-controls-js-2022-08-08-10-36-39-pages-localhost-NIST-800-53-module-Controls-index-html-2022-08-08-10_40_37
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-NIST-module-controls-js-2022-08-08-10-36-39-pages-localhost-NIST-800-53-module-Dashboard-index-html-2022-08-08-10_40_51
> 
> screencapture-file-home-mauri-repoPerformance-wazuh-kibana-app-test-performance-modules-sitespeed-result-NIST-module-controls-js-2022-08-08-10-36-39-pages-localhost-NIST-800-53-module-Events-index-html-2022-08-08-10_41_03
> 
>  </details>

**Pull request merged**